vitest.config.js

```diff
@@ -4,6 +4,10 @@ import { defineConfig } from "vite";
 export default defineConfig({
   test: {
     restoreMocks: true,
+    coverage: {
+      enabled: true,
+      reporter: ["text", "lcov", "html"],
+    },
```

Comment:
Can we enable coverage only for CI? When watching it prints a wall of coverage info, so have to keep scrolling up to see errors.